app/frontend/src/App.tsx

```diff
@@ -7,14 +7,22 @@ import Navigation from "./components/Navigation";
 import PageTitle from "./components/PageTitle";
 import { theme } from "./theme";
 
+const useStyles = makeStyles(theme => ({
+  offset: theme.mixins.toolbar,
+}))
+
 function App() {
+  const classes = useStyles();
   return (
     <Router>
       <ThemeProvider theme={theme}>
         <CssBaseline />
+        <Navigation />
+        <Hidden smDown>
+          <div className={classes.offset} />
```

Comment:
Maybe better use a Box. Why is this? To make space?

Reply:
Yes, it takes up the space of the fixed navigation. I've made a commit below to make this more clear, and use the existing `Container`.